rcl/src/rcl/arguments_impl.h

```diff
@@ -44,6 +44,11 @@ typedef struct rcl_arguments_impl_t
 
   /// Default log level (represented by `RCUTILS_LOG_SEVERITY` enum) or -1 if not specified.
   int log_level;
+  /// A file used to configure the external logging library
+  char * external_log_config_file;
+  bool log_stdout_disabled;
+  bool log_rosout_disabled;
+  bool log_ext_lib_disabled;
```

Comment:
nitpick: Even if a trivial sentence, it would be nice to have a doc brief for each of the new options.

Reply:
Added comments.